src/search-box/__tests__/search-box.spec.ts

```diff
@@ -88,16 +88,6 @@ describe('SearchBox', () => {
         expect(refine).toHaveBeenCalledTimes(0);
       });
 
-      it('should call refine when submit button has been clicked', () => {
-        const submitButton = fixture.debugElement.query(
-          By.css('button[type="submit"]')
-        );
-        submitButton.nativeElement.dispatchEvent(new Event('click'));
-
-        expect(refine).toHaveBeenCalledTimes(1);
-        expect(refine).toHaveBeenCalledWith('the query');
-      });
-
```

Comment:
test has been removed, since in current angular / jest / jest-preset-angular `button` no longer fires the submit event, and we can rely on browsers to do that